tests/test_trim_util.py

```diff
@@ -108,6 +110,7 @@ def test_do_trim_creates_an_lv(self,
                                    sr_get_capability,
                                    MockLock,
                                    lvutil):
+        lvutil._getVG_freespace.return_value = EMPTY_VG_SPACE
```

Comment:
Can you please let me know the reason here. As it is only for test use, i feel the current
value is sufficient.